src/nextcloud/talk/recording/Service.py

```diff
@@ -176,6 +179,10 @@ def start(self, actorType, actorId):
 
         fullDirectory = f'{directory}/{sanitizedBackend}/{self.token}'
 
+        # For linux/aarch64 aka arm64, the automatic finding mechanism of the executable will fail (unsupported by selenium in the specified version)
+        # Solution: make it configurable.
```

Comment:
Comment lines should be at most 80 characters wide (even if there are some code lines that are wider). Independently of that the comment looks better suited for a commit message explaining the change (or the configuration setting as described above).